articles/sharing_settings.md

```diff
@@ -8,23 +8,37 @@ order: 2
 
 ## Access Control
 
-Synapse has two ways to control who can access your content. `Sharing Settings`, or permissions, determine _who_ can access content and at what level. `Conditions for Use` define _how_ users with access may use the data. All content in Synapse has `Sharing Settings` but not all content has `Conditions for Use`. See the [Conditions for Use](/articles/access_control.html) article for information about those settings. This article covers sharing settings only.
+Synapse has two ways to control who can access your content. `Sharing Settings`, or permissions, determine _who_ can access content and at what level. `Conditions for Use` define _how_ users with access may use the data. All content in Synapse has Sharing Settings but not all content has Conditions for Use. See the [Conditions for Use](/articles/access_control.html) article for information about those settings. This article covers sharing settings only.
 
 <a name="sharing-setting"></a>
 
 ## Sharing Settings
 
-You can use the `Sharing Settings` to grant different levels of access to individuals or `Teams`. Often, users leverage these features to invite people to view or collaborate on a Synapse `Project`. Here are the kinds of access currently supported:
+You can use the `Sharing Settings` to grant different levels of access to individuals or `Teams`. Often, users leverage these features to invite people to view or collaborate on a Synapse `Project`. Sharing Settings can apply to an entire project or for individual files or folders within a project.
 
-* Can view
-* Can download
-* Can edit
-* Can edit & delete
-* Administrator
+There are two primary groups: **Public** and **Private**. The `Private` sharing setting limits access to only specified users and teams. By default, new projects are `Private`. The `Public` sharing setting applies to users on the web and any registered, logged-in Synapse user.
```

Comment:
Clarify the public sharing settings since it applies to either groups, but doesn't have to apply to both.

Reply:
Going to file a separate issue so we can close out our milestone. Thanks!